YoCode/Program.cs

```diff
@@ -1,23 +1,53 @@
 ﻿using System;
+using System.Linq;
 using System.IO;
 
 namespace YoCode
 {
     public class Program
     {
+
         static void Main(string[] args)
         {
+            var modifiedTestDirPath = args[0];
+            var originalTestDirPath = args[1];
+
+            var consoleOutput = new PrintToConsole();
+            var testResults = new TestResults();
+            var fileReader = new FileImport();
+
+            var modifiedTest = fileReader.GetAllFilesInDirectory(modifiedTestDirPath);
+            var originalTest = fileReader.GetAllFilesInDirectory(originalTestDirPath);
+
+            var dir = new Directory(originalTest, modifiedTest);
+
+            if (FileChangeChecker.ProjectIsModified(dir))
+            {
+                testResults.AnyFileChanged = true;
+                // UI test
+                var keyWords = new string[] { "miles", "kilometers", "km" };
+                var modifiedHtmlFiles = dir.GetFilesInDirectory(modifiedTestDirPath, FileTypes.html).ToList();
+
+                UICheck uiChecker = new UICheck(modifiedHtmlFiles, keyWords);
+
+                testResults.Lines = uiChecker.ListOfMatches;
+
+                // Solution file exists
+                testResults.SolutionFileExist = dir.GetFilesInDirectory(modifiedTestDirPath, FileTypes.sln).Count() != 0;
 
-            Console.WriteLine("Welcome to YoCode!");
+                // Git repo used
+                GitCheck gitChecker = new GitCheck();
+                gitChecker.ExecuteTheCheck();
+                testResults.GitUsed = gitChecker.GitUsed;
+                // Will not work on different systems due to issues in GitUsed method
```

Comment:
If you're going to use comments to mention things that need sorting out, it's better have a consistent style that you can easily search for. That way you can do a find-in-files when you think you're finished, to check if you've missed anything.

If you use `// TODO: <whatever>` as your style, then those comments will also show up in the Task List in Visual Studio. You can get to window from the View menu:
![image](https://user-images.githubusercontent.com/1788792/42390361-84afd056-8143-11e8-8725-fe21be1b79d8.png)

If thing you want to write the comment about won't be fixed for a while, then it's better to put it into your issue tracker in some way (that would be GitHub for us here). Either by adding a new item, or adding it as a task to an existing issue. 

GitHub issues can have [task lists](https://help.github.com/articles/about-task-lists/), which I find really handy for stuff like this.